src/components/decimal/decimal.test.js

```diff
@@ -255,6 +255,33 @@ context("Tests for Decimal component", () => {
         });
     });
 
+    it.each([
+      ["10%", "10%"],
+      ["30%", "30%"],
+      ["50%", "50%"],
+      ["80%", "80%"],
+      ["100%", "100%"],
+    ])(
+      "should use %s as maxWidth and render it with correct corresponding max-width percentage",
+      (input, totalWidth) => {
+        CypressMountWithProviders(<Decimal maxWidth={input} />);
+
+        getDataElementByValue("input")
+          .parent()
+          .parent()
+          .should("have.css", "max-width", totalWidth);
+      }
+    );
+
+    it("as maxWidth has no value it should render as 100%", () => {
```

Comment:
```suggestion
    it("when maxWidth has no value it should render as 100%", () => {
```

this phrasing makes more sense to me